### PR TITLE
docs(readme): add granian as an alternative

### PR DESCRIPTION
**Pull request by Zerotask**
# Summary

- Update mangum website; current website just shows a blank page
- Add Granian as an alternative

# Checklist

- [x] I understand that this PR may be closed in case there was no previous discussion. (This doesn't apply to typos!)
- [ ] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [x] I've updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,12 +131,17 @@ $ hypercorn app:App
 
 [Mangum][mangum] is an adapter for using ASGI applications with AWS Lambda & API Gateway.
 
+### Granian
+
+[Granian][granian] is a Rust HTTP server for Python applications which supports ASGI/3, RSGI and WSGI, HTTP/1 and HTTP/2, HTTPS and Websockets.
```
Comment (Kludex) [inline README.md]:
```suggestion
[Granian][granian] is a Rust HTTP server for Python applications which supports ASGI/3, RSGI and WSGI, HTTP/1 and HTTP/2, HTTPS and WebSockets.
```

**Review by Kludex — APPROVED** [review]
@gi0baro You good with this description?

File: README.md [hunk 2]
```diff
@@ -131,12 +131,17 @@ $ hypercorn app:App
 
 [Mangum][mangum] is an adapter for using ASGI applications with AWS Lambda & API Gateway.
 
+### Granian
+
+[Granian][granian] is a Rust HTTP server for Python applications which supports ASGI/3, RSGI and WSGI, HTTP/1 and HTTP/2, HTTPS and WebSockets.
```
Comment (gi0baro) [inline README.md]:
```suggestion
[Granian][granian] is an ASGI compatible Rust HTTP server which supports HTTP/2, TLS and WebSockets.
```

**Review by gi0baro — COMMENTED** [review]
@Kludex looks a bit *polluted*, left an alternative suggestion

**Comment by Kludex** [comment]
> @Kludex looks a bit *polluted*, left an alternative suggestion

Grazie 🙏

**Pull request merged**